src/java.xml/share/classes/javax/xml/validation/SchemaFactoryConfigurationError.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Hans, this looks like an oversight - the copyright year update should have retained the 2013 and additionally introduced 2024.

Reply:
Thanks for catching this, @jaikiran!